src/test/ui/borrowck/borrowck-describe-lvalue.rs

```diff
@@ -302,7 +302,7 @@ fn main() {
         // FIXME(#49824) -- the free region error below should probably not be there
         let mut x = 0;
            || {
-               || { //[mir]~ ERROR unsatisfied lifetime constraints
+               || { //[mir]~ ERROR captured variable cannot escape `FnMut` closure body
```

Comment:
i have to admit that I don't understand what's happening in this test. I.e. I don't even get why its failing.

Reply:
Or is my confusion exactly what the FIXME there is referring to? I grok that we cannot have multiple mutable borrows; but it’s the error about the captured variable escaping that I find confusing

Reply:
oh, okay, the new diagnostic explains this quite clearly. 😄